test-consumer/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+  `java-library`
+}
+
+dependencies {
```

Comment:
What exactly are we trying to showcase in this test?

Reply:
mostly this is to allow the dependency check scan to work on this repo. By including every reference directly in a local project, if any of them have vulnerabilities it'll be flagged in the BOM PR. Without this, scan won't pick up just constraints.

Reply:
Got it. So, the onus is on the person touching the platform boms to ensure the same dependency is added here.

Reply:
Yeah, for adding new dependencies. Version changes would not require anything here. If it's missed it's not like anything will break, but if the version is vulnerable in the bom it wouldn't be caught until it's attempted to be consumed.

So we'd basically have the following use cases:

- On new dependency
	- Add to catalog toml
	- Add to bom (using catalog reference)
	- Add to to test project (using catalog reference)
- On version update
	- Update version in catalog toml